KachnaOnline.Business/Facades/EventsFacade.cs

```diff
@@ -0,0 +1,186 @@
+        /// </summary>
+        /// <param name="from">If not null, only returns events with their from date being after or equal to the specified value.</param>
+        /// <param name="to">If not null, only returns events with their to date being before or equal to the specified value.</param>
+        /// <returns>A List of <see cref="KachnaOnline.Dto.Events.EventDto"/> or an empty list if no there is no event planned.</returns>
+        public async Task<IEnumerable<EventDto>> GetEvents(DateTime? from = null, DateTime? to = null)
+        {
+            var events = await _eventsService.GetEvents(from, to);
+            if (events is { Count: > 0 })
+            {
+                if (this.IsUserEventsManager())
+                    return events.Select(e => _mapper.Map<ManagerEventDto>(e)).ToList();
+
+                return events.Select(e => _mapper.Map<EventDto>(e)).ToList();
+            }
+
+            return new List<EventDto>();
+        }
+
+        /// <summary>
+        /// Returns an event with the given ID.
+        /// </summary>
+        /// <param name="eventId">ID of the <see cref="KachnaOnline.Dto.Events.EventDto"/> to return.</param>
+        /// <returns>An <see cref="KachnaOnline.Dto.Events.EventDto"/> with the given ID.</returns>
+        /// <exception cref="EventNotFoundException"> thrown when an event with the given
+        /// <paramref name="eventId"/> does not exist.</exception>
+        public async Task<EventDto> GetEvent(int eventId)
+        {
+            var returnedEvent = await _eventsService.GetEvent(eventId);
+
+            if (this.IsUserEventsManager())
+                return _mapper.Map<ManagerEventDto>(returnedEvent);
+
+            return _mapper.Map<EventDto>(returnedEvent);
+        }
+
+        /// <summary>
+        /// Returns a list of next events.
+        /// </summary>
+        /// <returns>A List of <see cref="KachnaOnline.Dto.Events.EventDto"/> planned as the next events or an empty list if no event is planned.
+        /// The list usually contains only one event, more in case multiple next events start at the same time.</returns>
+        public async Task<IEnumerable<EventDto>> GetNextPlannedEvents()
+        {
+            var events = await _eventsService.GetNextPlannedEvents();
+            if (events is { Count: > 0 })
+            {
+                if (this.IsUserEventsManager())
+                    return events.Select(e => _mapper.Map<ManagerEventDto>(e)).ToList();
+
+                return events.Select(e => _mapper.Map<EventDto>(e)).ToList();
+            }
+
+            return new List<EventDto>();
+        }
+
+        /// <summary>
+        /// Plans a new event.
+        /// </summary>
+        /// <param name="newEvent">New event data to create the new event with.</param>
+        /// <returns>Created event with its ID attribute filled.</returns>
+        public async Task<ActionResult<ManagerEventDto>> PlanEvent(BaseEventDto newEvent)
```

Comment:
Facade should probably not return `ActionResult`. This is probably what's causing a weird return value (the result, value should not be there):
```
{
  "result": null,
  "value": {
    "madeById": 704,
    "id": 1,
    "name": "Start@FIT",
    "place": "VUT BUT",
    "placeUrl": "https://www.placeurl.com",
    "imageUrl": "https://www.imageurl.com",
    "shortDescription": "Meet the faculty.",
    "fullDescription": "string",
    "url": "https://www.url.com",
    "from": "2022-09-18T12:30:00",
    "to": "2022-09-20T19:00:00",
    "linkedPlannedStateIds": []
  }
}
```

Reply:
I fixed this already once, but somehow lost the change somewhere in the process. Changed to simple `ManagerEventDto` again.